### ISSUE TITLE
Typescript not compiling to JS after webpack

### ISSUE THREAD

**Issue by kmuthukrishnan** [NativeScript/nativescript-dev-webpack#244]
Hi,

I created a Angular2 Nativescript hello world project. I made few changes to the style and saved these. The ts file changes reflected correctly in the js files.

I then implemented webpack with AOT. The package is correctly building on “npm run start-ios-bundle”, and the app displays on the emulator

However, now I find that any changes made to the ts files are no longer appearing in the js files on saving or even after build. To solve this, I tried including “compileOnSave”: true in tsconfig.json, but that is not helping.

How do I solve this issue?

**Comment by NickIliev** [comment]
@kmuthukrishnan try to manually transpile the files by executing the `tsc` command.
If that does not help please post the content of your package.jjson file and describe the steps you are taking when working with your project.
Also. check if you have by any chance excluded project folders in your `tsconfig.json` - more info about it [here](http://docs.nativescript.org/angular/best-practices/bundling-with-webpack.html#angular-and-ahead-of-time-compilation)

**Comment by sis0k0** [comment]
@kmuthukrishnan,

I assume that you are using the default webpack config provided by the nativescript-dev-webpack plugin. When you are building with nativescript-cli (`tns build android`, for example), the .ts files are transpiled and the newly created .js files are stored on the disk (hard drive, ssd, etc.) at the same location as the corresponding .ts files.
However, when you are building with webpack (`npm run build-android-bundle`), the TypeScript files are transpiled by the `@ngtools/webpack` plugin. More specifically, that plugin uses a private API from `ngc` (the Angular compiler), which actually calls `tsc-wrapped` - an Angular wrapper module around the TypeScript compiler. So, with that said, when you are bundling your app with webpack, `tsc-wrapped` transpiles your .ts files, but it doesn't save them on the disk. During the webpack build, the transpiled JavaScript code is stored in-memory (in your RAM), rather than on the physical disk. In the final phase of the webpack build, the code is moved inside two JS files (in the most common scenario) - `bundle.js` (your app code) and `vendor.js` (your node_modules). These two files are stored in your `platforms/ios/NAME_OF_YOUR_APP_/app` folder for iOS, and in your `platforms/android/src/main/assets/app` folder for Android.
So, basically, that's why you won't see any difference in the .js files that are stored next to the .ts files in the '/app' folder. Still, your changes will be reflected on the app.
